### ISSUE TITLE
config.go comments

### ISSUE THREAD

**Issue by zakird** [zmap/zgrab2#8]
```
	MetaFileName       string          `short:"m" long:"metadata-file" default:"-" description:"Metadata filename, use - for stdout"`
	LogFileName        string          `short:"l" long:"log-file" default:"-" description:"Log filename, use - for stdout"`
```
`-` should redirect to stderr not stdout for both of these 

```
func init() {
	config.Multiple.ContinueOnError = true //set default for multiple value
}
```
Why isn't this just a command-line argument

```
	switch config.LogFileName {
	case "-":
		config.logFile = os.Stderr
	default:
		var err error
		if config.logFile, err = os.Create(config.LogFileName); err != nil {
			log.Fatal(err)
		}
		log.SetOutput(config.logFile)
	}
```
This is confusing logic as the non-default logic is labeled as default. Why not just use an `if` statement since there are only two options and that more clearly explains your logic?

```
if config.ConnectionsPerHost > 50 || config.ConnectionsPerHost < 1 {
```

You just checked that second predicate a line earlier.

**Comment by ajholland** [comment]
config.Multiple.ContinueOnError is a command-line argument, we set it's default to true because that's what makes sense for the option and because this flags library doesn't support setting defaults for boolean values other than false because supposedly that doesn't make sense. I can look into changing that/digging into it.

:+1: on everything else

**Comment by ajholland** [comment]
Resolved by [424ee8c](https://github.com/zmap/zgrab2/commit/424ee8c071b89704e424dce80a5147bbddb60b4f)
